.github/workflows/build.yml

```diff
@@ -198,6 +211,20 @@ jobs:
         run: cmake --build build --target tests
       - name: Package
         run: cmake --build build --target package
+      - name: Unpack
+        run: |
+          mkdir build/install
+          7z x -obuild/install build/lmms-*.exe
```

Comment:
Instead of unpacking the installer, it should be possible to run LMMS from whatever staging directory CPack uses to collect the files to install in the first place. Alternatively, `cmake --install build --prefix build/install` should do the same thing (or an equivalent command for the older CMake version used for the MinGW builds).

Reply:
@DomClark is your comment only valid for this code section or also for MSVC, where we also have the pack/unpack steps?

Reply:
Having thought about it some more, I think it's better to unpack the installer rather than relying on internal behaviour of CMake/CPack. This also helps ensure that the installer contains all the necessary files to run LMMS. However, you could use the 7Zip approach for MSVC builds too (the runners have it preinstalled), avoiding the need to handle different Program Files locations.

Reply:
Good idea. I did that, now the matrix does not need to be that complex anymore and the paths are unique, as you predicted.